spyder_kernels/comms/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2009- Spyder Kernels Contributors
+#
+# Licensed under the terms of the MIT License
+# (see spyder_kernels/__init__.py for details)
+# -----------------------------------------------------------------------------
+
+"""
+Comms
```

Comment:
```suggestion
Let's add something like:

```
Comms allow custom messages between the frontend and the kernel.


They are used, for instance, in ipywidgets to update widget state.

A comm consists of a pair of objects, in the kernel and the frontend, with an automatically assigned unique ID. When one side sends a message, a callback on the other side is triggered with that message data. Either side, the frontend or kernel, can open or close the comm.

See: https://jupyter-notebook.readthedocs.io/en/stable/comms.html
```

Reply:
Well what I call comms here are wrappers around the jupyter comms, not the comms themselves. the jupyter comms are stored in CommsBase._comm

Reply:
Hmm ok, but let's add something to that docstring. Feels a bit bare bones :-p

Reply:
@impact27, you can just add what you put in your comment above. That would be fine.